porting/index.md

```diff
@@ -62,15 +69,54 @@ You can also specify `-j[num]` as it will fetch the files simultaneously. This d
 
 ### Put parts together
 
-Now you need to put all the parts for your device together, and since our tree is based on LineageOS for `halium-7.1` or CyanogenMod for `halium-5.1` you can use the device files they provide.
+Now you need to put all the parts for your device together, and since our tree is based on LineageOS for `halium-7.1` or CyanogenMod for `halium-5.1` you can use the device files they provide. Take care to switch the branch selection in Githubs webpage so you download the correct branches!
```

Comment:
Ok will fix it. I saw now that of course its necessary to specify the correct branch in the revision part of the manifest xml...